docs/contributing.rst

```diff
@@ -35,7 +35,7 @@ Splitting Pull Requests
 
 If you discussed your feature within an issue (as advised), there's a great
 chance that the implementation appears doable in several steps. In order to
-facilitate the review process, we strongly advise to split your feature
+expedite the review process, we strongly advise to split your feature
 implementation in small pull requests (if that is possible) so they contain a
 very small number of commits (a single commit per pull request being optimal).
```

Comment:
As someone probably less familiar with this process, I'm not sure I entirely understand it. You need to make a separate PR for each commit? For example, when doing Mariadb, I feel like I just made one large PR. Was that incorrect?

Reply:
Yeah this is just the idea of keeping PRs focused which comes with many good properties (mergeability, revertability, readability, etc). For the latter part yeah that doesn't apply anymore since we squash-merge. When this section was written PRs were not squashed so you'd end up with a bunch of "fix typo" commits in the main branch.

Good catch! I'll remove this line :+1:ll